cms/search/tests/test_api_views.py

```diff
@@ -0,0 +1,132 @@
+from django.test import TestCase, override_settings
+
+from cms.articles.tests.factories import ArticleSeriesPageFactory, StatisticalArticlePageFactory
+from cms.home.models import HomePage
+from cms.methodology.tests.factories import MethodologyPageFactory
+from cms.release_calendar.models import ReleaseCalendarIndex
+from cms.release_calendar.tests.factories import ReleaseCalendarPageFactory
+from cms.search.tests.helpers import ExternalAPITestMixin, ResourceDictAssertions
+from cms.standard_pages.tests.factories import IndexPageFactory, InformationPageFactory
+from cms.themes.tests.factories import ThemePageFactory
+from cms.topics.tests.factories import TopicPageFactory
+
+RESOURCE_ENDPOINT = "/v1/resources/"
+
+
+@override_settings(IS_EXTERNAL_ENV=False)
+class SearchResourcesViewTests(TestCase, ResourceDictAssertions, ExternalAPITestMixin):
+    @classmethod
+    def setUpTestData(cls):
+        # Pages that are excluded from the search index
+        cls.excluded_pages = [
+            ArticleSeriesPageFactory(),
+            HomePage(),
+            ReleaseCalendarIndex(),
+            ThemePageFactory(),
+            TopicPageFactory(),
+        ]
```

Comment:
Can you add a draft instance of one of the pages that would normally be included. We need a test that confirms that only _live_ pages are shown

Reply:
Done!